client/app/(tabs)/homeScreen.tsx

```diff
@@ -1,22 +1,20 @@
-import { View , Text} from 'react-native'
-import HomeStories from '@/components/Home/HomeStories';
-import FaveStories from '@/components/Home/FaveStories';
+import { View, Text } from 'react-native'
+import FaveStories from '@/components/Home/FaveStories' 
+import AllStories from '@/components/Home/AllStories'
 
 const homeScreen = () => {
 
 
   return (
-    <View className ='flex-1 bg-dark-grey'>
-    <View className='flex flex-col justify-center' >
-      <Text className='text-green text-3xl font-bold mb-4 mt-2 text-center'>All Stories</Text>
-      <HomeStories />
-      <Text className='text-green text-3xl font-bold mt-2 mb-4 text-center'>Favorite Stories</Text>
-      <FaveStories/>
-    </View>
+    <View className='flex-1 bg-dark-grey'>
+      <View className='flex flex-col' >
+        <Text className='text-green text-3xl font-bold mb-4 mt-3 text-center'>All Stories</Text>
```

Comment:
Should show create new story button if no stories available. Text should be moved into components